### PR TITLE
Wearing magboots will no longer send you flying when unwrenching pipes

### LINKED ISSUE #14813: Unwrenching gas pipe with magboot enabled still sending you fly away!!!
### Severity

Quality of Life

### Short Description

Magboot didn't stop you from getting blown by unwrenching gas pipe

### Reproduction

1. Join round as atmos tech
2. Fill TONS of gas in a pipe
3. Get magboot and enabled it
4. unwrench gas pipe
5. YOU GET SENT TO BRAZIL!!

### Round ID

Round #41833 or every rounds

### Test Merges

_No response_

### BYOND Username

Warface1234455

### Additional information

Unga bunga

### PR DESCRIPTION
**Pull request by redmoogle**
# Document the changes in your pull request

Closes: #14813

having magboots equipped and having them on will no longer launch you (or being a cyborg)

mostly just a QOL feature

# Wiki Documentation

Magboots help you not get thrown away

# Changelog

:cl:  
rscadd: having magboots equipped and having them on will no longer launch you when unwrenching pipes
/:cl:

### CODE REVIEW

File: code/modules/atmospherics/machinery/atmosmachinery.dm [hunk 1]
```diff
@@ -233,6 +233,13 @@ GLOBAL_LIST_EMPTY(pipeimages)
 		var/datum/gas_mixture/env_air = loc.return_air()
 		pressures = int_air.return_pressure() - env_air.return_pressure()
 
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		var/obj/item/clothing/shoes/magboots/magboot = H.get_item_by_slot(SLOT_SHOES)
+		if(magboot && istype(magboot) && magboot.magpulse)
+			return // Dont send you flying
+		
+
```
Comment (Hopekz) [inline code/modules/atmospherics/machinery/atmosmachinery.dm]:
```suggestion
```

Comment (redmoogle) [reply code/modules/atmospherics/machinery/atmosmachinery.dm]: im murdering you

Comment (Hopekz) [reply code/modules/atmospherics/machinery/atmosmachinery.dm]: > im murdering you

added to the evidence I'll leave behind if someone murders me

**Comment by warface1234455** [comment]
I think its the same proc

**Comment by redmoogle** [comment]
> I think its the same proc

what

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**